systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBasicExampleClients.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.crd.kafkaclients;
+
+import io.fabric8.kubernetes.api.model.batch.DoneableJob;
+import io.fabric8.kubernetes.api.model.batch.JobBuilder;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.resources.KubernetesResource;
+import io.strimzi.systemtest.resources.ResourceManager;
+import io.strimzi.systemtest.utils.ClientUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.security.InvalidParameterException;
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class KafkaBasicExampleClients {
+
+    private static final Logger LOGGER = LogManager.getLogger(KafkaBasicExampleClients.class);
+
+    protected String producerName;
+    protected String consumerName;
+    protected String bootstrapServer;
+    protected String topicName;
+    protected int messageCount;
+    protected String additionalConfig;
+    protected String consumerGroup;
+    protected long delayMs;
+
+    public abstract static class Builder<T extends Builder<T>> {
+        private String producerName;
+        private String consumerName;
+        private String bootstrapServer;
+        private String topicName;
+        private int messageCount;
+        private String additionalConfig;
+        private String consumerGroup;
+        private long delayMs;
+
+        public T withProducerName(String producerName) {
+            this.producerName = producerName;
+            return self();
+        }
+
+        public T withConsumerName(String consumerName) {
+            this.consumerName = consumerName;
+            return self();
+        }
+
+        public T withBootstrapServer(String bootstrapServer) {
```

Comment:
Is this just a server or address? I do not see any `withBootstrapPort` ... so maeby naming it `withBootstrapAddress` would be better to make it more clear it is server+port?

Reply:
Sure, sounds good, thanks :)